04_beerhouse/src/MainPage.jsx

```diff
@@ -0,0 +1,208 @@
+import styled from 'styled-components';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import BeerHouse from './BeerHouse';
+
+const getLocation = (errHandler) => {
+  if ('geolocation' in navigator) {
+    return new Promise((resolve) => {
+      navigator.geolocation.getCurrentPosition(
+        (position) => {
+          const {
+            coords: { latitude: y, longitude: x },
+          } = position;
+          resolve({ x, y });
+        },
+        (e) => {
+          alert('HTTPS 연결을 확인해주세요.');
+          errHandler && errHandler();
+        }
+      );
+    });
+  }
+
+  return { x: 126.8350976, y: 37.617664 };
+};
+
+async function 위치가져오기() {
+  const result = await getLocation();
+  const { x, y } = result;
+  return { x, y };
+}
+
+async function 내근처맥주집가져오기() {
+  const currentLocation = await 위치가져오기();
+  const result = await axios.get(
+    'https://dapi.kakao.com/v2/local/search/keyword',
+    {
+      headers: {
+        Authorization: `KakaoAK ${process.env.REACT_APP_KAKAO_AK}`,
+      },
+      params: {
+        x: currentLocation.x,
+        y: currentLocation.y,
+        radius: 1000,
+        query: '맥주',
+      },
+    }
+  );
+  return result.data.documents;
+}
+
+async function 특정지역맥주집가져오기(location = '혜화') {
+  const result = await axios.get(
+    'https://dapi.kakao.com/v2/local/search/keyword',
+    {
+      headers: {
+        Authorization: `KakaoAK ${process.env.REACT_APP_KAKAO_AK}`,
+      },
+      params: {
+        // eslint-disable-next-line no-useless-concat
+        query: location + ' ' + '맥주',
+      },
```

Comment:
```suggestion
      params: {
        // eslint-disable-next-line no-useless-concat
        query: `${location} 맥주`,
      },
```
이렇게 써보면 어떨까? 이러면 eslint 주석 없애도 문제 없을 것 같아